### ISSUE TITLE
Unit tests aren't ran during PR's?

### ISSUE THREAD

**Issue by MTijn** [php-sepa-xml/php-sepa-xml#158]
Morning,

When I opened a PR I did see that you check for code readability when code is altered, but I could not find any unit tests being run in the code climate pages.

Would it be an idea to have these running in Github Actions?

**Comment by BorislavSabev** [comment]
Yes, for sure. If you can push a PR to that avail it would be great.

**Comment by BorislavSabev** [comment]
Fixed
